manifest.json

```diff
@@ -1,11 +1,12 @@
 {
     "files": {
-        "index.js": "610cc881f98413c6aa1c7229f87c6beeda43553fa4c5dc53bf4259b514a0eef6",
+        "README.md": "e51994e26e7cee937cf6df94a14783dc4e51be880aaa1d848befef1464878c05",
+        "index.js": "0fd872a2e689b0adf7886303e09483ebdcb56c8b7bfefd76ea3aa7ae4a658ebf",
         "module.json": "632e8ae936ebc7e300590b4c7671c3e4cab3279ee49229d7500e2cb054ca3095"
     },
     "defs": {
         "C_DEL_ITEM": 2,
-		"S_INVEN": [17,18]
-        "S_LOGIN": 12,
```

Comment:
You don't add command at the end of the line here, it's the last element of this object, if it's javascript file, it may be ignored if you did this as it will get auto corrected during read, but it's json file, it will lead to error.